src/app/components/media/MediaComments.js

```diff
@@ -105,6 +107,7 @@ const MediaCommentsContainer = Relay.createContainer(withPusher(MediaCommentsCom
     eventTypes,
     fieldNames,
     annotationTypes,
+    teamSlug: /^\/([^/]+)/.test(window.location.pathname) ? window.location.pathname.match(/^\/([^/]+)/)[1] : null,
```

Comment:
This is extracted when the `index.js` bundle is parsed. It won't update if the user switches teams (I don't think).

How about adding `teamSlug` in `prepareVariables()`? https://relay.dev/docs/en/classic/classic-api-reference-relay-container#preparevariables

```
prepareVariables: vars => ({ ...vars, teamSlug: /.../... }),
```